httpie/output/formatters/xml.py

```
@@ -0,0 +1,61 @@
+    In-place prettyprint formatter
+    C.f. http://effbot.org/zone/element-lib.htm#prettyprint
+
+    """
+    def _indent(elem, level=0):
+        i = "\n" + level * indent_text
+        if len(elem):
+            if not elem.text or not elem.text.strip():
+                elem.text = i + indent_text
+            if not elem.tail or not elem.tail.strip():
+                elem.tail = i
+            for elem in elem:
```

Ruff (0.8.2)

Loop control variable `elem` overrides iterable it iterates

(B020)